src/styles/Login.css

```diff
@@ -1,5 +1,15 @@
-body {
-  background: #b93c3c;
+.already {
```

Comment:
Could it be a good idea to add variables if the same color is used several times? (eg. `#f50d51`)
cf. [(https://www.w3schools.com/css/css3_variables.asp)](https://www.w3schools.com/css/css3_variables.asp)

Reply:
And FYI, MUI uses inline css. 
Open question: could it be more convenient when working on components that have a "unique" style (not used elsewhere) instead or creating another `.css` file?